grpcdogfood/server/dogfood_server.go

```diff
@@ -0,0 +1,50 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2021 Datadog, Inc.
+
+package main
+
+import (
+	"context"
+	"fmt"
+	"log"
+	"net"
+
+	pb "github.com/DataDog/chaos-controller/grpcdogfood/chaosdogfood"
+	"google.golang.org/grpc"
+	emptypb "google.golang.org/protobuf/types/known/emptypb"
+)
+
+const serverAddr = "localhost:50051"
+
+type chaosDogfoodServer struct {
+	pb.UnimplementedChaosDogfoodServer
+}
+
+func (s *chaosDogfoodServer) Order(ctx context.Context, req *pb.FoodRequest) (*pb.FoodReply, error) {
+	fmt.Println("* " + req.Animal + " food ordered")
+	return &pb.FoodReply{Message: "Mock Reply", ConfirmationId: 1}, nil
+}
+
+func (s *chaosDogfoodServer) GetCatalog(ctx context.Context, req *emptypb.Empty) (*pb.CatalogReply, error) {
+	fmt.Println("* catalog delivered")
+	return &pb.CatalogReply{Items: []*pb.CatalogItem{}}, nil
+}
+
+func main() {
+	fmt.Println("connecting to " + serverAddr + "...")
```

Comment:
Nit: `"listening on serverAddr..."` would be clearer. Also, prefer `fmt.Printf("listening on %v...\n", serverAddr)`.

Also update the setup.md example output.